package.json

```diff
@@ -35,12 +35,14 @@
   "homepage": "https://github.com/twilio/twilio-webrtc.js#readme",
   "devDependencies": {
     "browserify": "^14.4.0",
+    "electron": "^5.0.0",
     "envify": "^4.1.0",
     "eslint": "^4.4.1",
     "istanbul": "^0.4.5",
     "karma": "^1.7.0",
     "karma-browserify": "^5.1.1",
     "karma-chrome-launcher": "^2.2.0",
+    "karma-electron": "git://github.com/twilio/karma-electron.git#1.0.0-alpha1",
```

Comment:
@makarandp0 Dependencies should not have a `git` url because you should not need permissions for npm to clone the tag. Can you change it to:
```
github:twilio/karma-electron#tag_name
```
?

Reply:
good point, Thanks! I will update the dependency and make sure it works.